src/client/context.rs

```diff
@@ -818,15 +907,18 @@ impl Context {
         rest::edit_note(user_id.into().0, map)
     }
 
+    /// Get a vector of bans server has.
     pub fn get_bans<G: Into<GuildId>>(&self, guild_id: G) -> Result<Vec<Ban>> {
         rest::get_bans(guild_id.into().0)
     }
 
+    /// Get all [`RichInvite`]s a channel has.
```

Comment:
ditto about the brackets and markdown link, it should just be:
```rs
/// get all `RichInvite`s a channel has.
```
or maybe since RichInvite is in the signature, it'd sound nicer as just "Get all invites a channel has."?